
<html>
  <head>
    <title>Lab 05 - Graded Solution</title>
  </head>
  <body>
    <h1>Lab 05 - Graded Solution</h1>
    <p><b>You scored 78% = 2.34 points for this lab.</b></p>
    <ul>
      <li><b>test_subscript_access</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.097 sec</Li>
      </ul>
      </li>
      <li><b>test_custor_based_access</b>
      <ul>
      <li>Grade Percentage: 12</li>
      <li>Your solution succeeded taking 0.122 sec</Li>
      </ul>
      </li>
      <li><b>test_stringification</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.098 sec</Li>
      </ul>
      </li>
      <li><b>test_single_element_manipulation</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.097 sec</Li>
      </ul>
      </li>
      <li><b>test_predicates</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.099 sec</Li>
      </ul>
      </li>
      <li><b>test_queries</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution failed this test case! taking 0.141 sec</Li>
<li><pre>ValueError: not found</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 48, in wrapper
    result = test_function(module, *args, **kwargs)
  File "/var/folders/f7/xnz8v_b13td37s3k4w8t5j2h0000gn/T/tmpzxtit2vt/lab05-test.py", line 241, in test_queries
    assertEquals(7, lst.index(2, 4, -1))
  File "<tested-program>", line 274, in index
ValueError: not found
</pre></li>
      </ul>
      </li>
      <li><b>test_bulk_operations</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.099 sec</Li>
      </ul>
      </li>
      <li><b>test_iteration</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution succeeded taking 0.099 sec</Li>
      </ul>
      </li>
      <li><b>test_reverse</b>
      <ul>
      <li>Grade Percentage: 11</li>
      <li>Your solution failed this test case! taking 0.097 sec</Li>
<li><pre>TypeError: unsupported operand type(s) for //: 'list' and 'int'</pre></li>
<li><pre>Traceback (most recent call last):
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/execution_base.py", line 60, in call_test_function
    test_function(
  File "/Users/lord_pretzel/.pyenv/versions/3.9.0/lib/python3.9/site-packages/grader-0.0.1-py3.9.egg/grader/core.py", line 48, in wrapper
    result = test_function(module, *args, **kwargs)
  File "/var/folders/f7/xnz8v_b13td37s3k4w8t5j2h0000gn/T/tmp00xnee8o/lab05-test.py", line 333, in test_reverse
    rev = lst.reverse()
  File "<tested-program>", line 346, in reverse
TypeError: unsupported operand type(s) for //: 'list' and 'int'
</pre></li>
      </ul>
      </li>
    </ul>
  </body>
</html>
